containers/python-examples/docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: "3.4"
+
+services:
+  grpc-api:
+    image: ghcr.io/deephaven/grpc-api:${VERSION:-latest}
+    expose:
+      - '8080'
+    volumes:
+      - ./docker/core/data:/data
+      - api-cache:/cache
+    environment:
+      - IO_DEEPHAVEN_CONSOLE_TYPE=python
```

Comment:
This is not present in `python`.  It raises questions about what is different in the files and why.  Have you done diffs of the files to look for differences?

Reply:
Fixed!